### DISCUSSION TITLE
debian-buster:1.8.0 source?

### DISCUSSION THREAD

**Discussion by Riya-Pallikala-LM** [signalfx/splunk-otel-collector#3412] [Q&A] [answered]
The latest version of this repo requires ci-container/debian-buster:1.8.0, but I'm not able to find that in hub.docker.com. Can someone advise where this image can be obtained?

**Comment by jeffreyc-splunk** [comment, marked as answer]
@Riya-Pallikala-LM Assuming you are referring to https://github.com/signalfx/splunk-otel-collector/blob/main/.gitlab-ci.yml#L2, this is an image we use for our gitlab pipelines from an internal repository.
